### ISSUE TITLE
AMOC country temperatures scale weirdly and don't seem to pass on to consumption

### ISSUE THREAD

**Issue by felixschaumann** [openmodels/META-2021#8]
Hi! While playing around with the AMOC component, I noticed two things:

1) `T_country_AMOC` values get huge.
2) `T_country_AMOC` values don't impact the SCC.

I think issue 1) lies in this line:
https://github.com/openmodels/META-2021/blob/70e4bd9ceeb5e17f14b14f4ebf0e465e3dd5421d/src/MimiMETA.jl#L167
Instead of scaling with `:PatternScaling=>:scale_country`, the AMOC components scales with `:PatternScaling=>:T_country`.

Issue 2), as far as I can see, arises because `T_country_AMOC` is not passed on to `Consumption`.

I tried to fix that by replacing the line cited above with:
```julia
disconnect_param!(model, :Consumption, :T_country)
connect_param!(model, :AMOC=>:scale_country, :PatternScaling=>:scale_country);
connect_param!(model, :Consumption=>:T_country, :AMOC=>:T_country_AMOC);
```
Not entirely sure whether disconnecting `T_country` from `Consumption` is necessary, but I wanted to be on the safe side.

Cheers and thanks for developing and open-sourcing this great model!

**Comment by jrising** [comment]
Hi @felixschaumann! Thanks for looking into this.

First, I think that you are right about both of your points. I checked our testing of this, which we do at the component-level and at the system-level. At the component-level, we do not run into problem 1 because we pass in the correct scaling. At the system-level, we miss problem 2 because AMOC has a fairly small effect on global welfare and no effect on the other variables we check.

I want to test your fixes, but they look right to me.

**Comment by jrising** [comment]
This is resolved in PR #10. Sorry for not closing before.
